pkg/resources/server.go

```diff
@@ -271,17 +271,31 @@ func (rs *resourceServer) Start() error {
 			glog.Errorf("serving incoming requests failed: %s", err.Error())
 		}
 	}()
-	// Wait for server to start by launching a blocking connection
-	ctx, _ := context.WithTimeout(context.TODO(), serverStartTimeout)
-	conn, err := grpc.DialContext(
```

Comment:
new version of the grpc doesn't make a connection when creating the client so to maintain the same functionally here we need to create the client and run a connect function that is a blocking one